src/main/java/cpsc/dlsproject/ast/Expressions/Values/BooleanValue.java

```diff
@@ -0,0 +1,33 @@
+package cpsc.dlsproject.ast.Expressions.Values;
+
+public class BooleanValue extends Value {
+    public boolean isValue() {
+        return value;
+    }
+
+    private boolean value;
```

Comment:
It's going outside of the convention, but let's stick to making the attributes for AST public. This will save us time than trying to unwrap the value via methods everytime we need to check the attributes.